dockstore-integration-testing/src/license/THIRD-PARTY.properties

```diff
@@ -0,0 +1 @@
+org.antlr--antlr-runtime--3.4=The BSD License (https://www.antlr.org/license.html)
```

Comment:
See previous comment asking if this can be moved to the root THIRD-PARTY.properties.

Reply:
I was unable to find a way to do this.